dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -714,6 +718,21 @@ private Response multipartPUTandPOST(HttpServletRequest request, HttpServletResp
                     .equals("json")) {
                 try {
                     processJSON(contentlet, part.getEntityAs(InputStream.class));
+                    try {
+                        binaryFieldsInput =
+                            webResource.processJSON(part.getEntityAs(InputStream.class)).get("binary_fields")
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)